pkg/target/transformersimage_test.go

```diff
@@ -283,3 +283,56 @@ spec3:
         name: my-cool-app
 `)
 }
+
+func makeTransfomersImageKnativeBase(th *KustTestHarness) {
+	th.writeK("/app/base", `
+resources:
+- knative.yaml
+configurations:
+- config/knative.yaml
+images:
+- name: solsa-echo
+  newTag: foo
+`)
+	th.writeF("/app/base/knative.yaml", `
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: echo
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: solsa-echo
+`)
+	th.writeF("/app/base/config/knative.yaml", `
+images:
+- path: spec/runLatest/configuration/revisionTemplate/spec/container
```

Comment:
This should be ` spec/runLatest/configuration/revisionTemplate/spec/container/image`.

Reply:
done